tests/score-calculator.spec.ts

```diff
@@ -71,3 +107,36 @@ describe('calculator', () => {
     ).to.throw();
   });
 });
+
+describe('Calculate correctly base scores', () => {
+  Object.entries(cvssTests).map((entry) => {
+    const cvss = entry[0];
+    const baseScore = entry[1][0];
+    it(`should calculate a score of ${baseScore} for ${cvss}`, () => {
+      const score = calculateBaseScore(cvss);
+      expect(score).to.equal(baseScore);
+    });
+  });
+});
+
+describe('Calculate correctly temporal scores', () => {
```

Comment:
If common `calculateScore()` method, mentioned in comment for `ScoreResult`, will be implemented - we can combine those three different suites into one